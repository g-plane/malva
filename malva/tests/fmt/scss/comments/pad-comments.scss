/*cfg formatComments = true*/
//comments
///doc comments
/// doc comments
