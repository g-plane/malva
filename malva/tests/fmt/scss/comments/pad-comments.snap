---
source: malva/tests/fmt.rs
---
/* cfg formatComments = true */
// comments
/// doc comments
/// doc comments
